msal4j-sdk/pom.xml

```diff
@@ -157,6 +157,12 @@
             <version>2.14.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>jakarta.xml.bind</groupId>
+            <artifactId>jakarta.xml.bind-api</artifactId>
```

Comment:
We're trying to remove reliance on dependencies as much as possible: https://github.com/AzureAD/microsoft-authentication-library-for-java/issues/909. Seems like this is only used in the new TokenRevocationUtil class, and shouldn't be needed if that class is removed (see my comment there).

Reply:
Removed